src/sagas/workspaces.ts

```diff
@@ -494,10 +495,19 @@ export function* evalCode(
     }
   }
 
+  const isNonDet: boolean = context.chapter === 4.3;
+  if (isNonDet) {
+    context.executionMethod = 'non-det-interpreter';
+  }
+
   const { result, interrupted, paused } = yield race({
     result:
       actionType === actionTypes.DEBUG_RESUME
         ? call(resume, lastDebuggerResult)
+        : code === 'try_again;'
```

Comment:
I think the exact command used for `try_again` should be kept as a constant in js-slang.
This will allow us to easily standardise the command in both the frontend and node repl